examples/example1d_SAM_ParametricStudy/README.md

```diff
@@ -0,0 +1,15 @@
+# Example1d - MOOSE/SAM Plugin
+
+## Purpose
+
+This example provides a demonstration on how to use WATTS to perform multiple runs and save selected outputs from all runs to a single '.csv' file for ease of comparison, post-processing, and visualization.
+
+## Code(s)
```

Comment:
As an improvement to the README documentation, I am thinking we should have here a list of files, and basic description, something like:
* example1d.py: WATTS workflow for this example, this is the file to execute to run the problem above-described
* sam_template: SAM templated input file
* results.csv: an example of csv file that will be generated through running this example

Reply:
I added a new section called 'File descriptions' to the README file as you suggested. Thanks.

Reply:
@zhieejhia93 I added links to the files in the README.

Reply:
Got it. The links are a good idea.